Source/WebCore/editing/cocoa/HTMLConverter.mm

```diff
@@ -2427,33 +2438,30 @@ static String preferredFilenameForElement(const HTMLImageElement& element)
     }
 #endif
 
-    RetainPtr fileWrapper = fileWrapperForElement(element);
-    RetainPtr attachment = adoptNS([[PlatformNSTextAttachment alloc] initWithFileWrapper:fileWrapper.get()]);
-    return [NSAttributedString attributedStringWithAttachment:attachment.get()];
-}
-
-static RetainPtr<NSAttributedString> attributedStringWithAttachmentForElement(const HTMLAttachmentElement& element)
-{
-    RetainPtr fileWrapper = fileWrapperForElement(element);
-    RetainPtr attachment = adoptNS([[PlatformNSTextAttachment alloc] initWithFileWrapper:fileWrapper.get()]);
-    return [NSAttributedString attributedStringWithAttachment:attachment.get()];
+    return attributedStringWithAttachmentForElement(element);
 }
 
 #if ENABLE(WRITING_TOOLS)
-static bool hasAncestorQualifyingForWritingToolsPreservation(Element* ancestor, WeakHashMap<Element, bool, WeakPtrImplWithEventTargetData>& cache)
+static bool hasAncestorQualifyingForWritingToolsPreservation(const Element* ancestor, WeakHashMap<Element, bool, WeakPtrImplWithEventTargetData>& cache)
```

Comment:
Also not a big fan of an entirely unrelated change being made here...

Reply:
undone